themes/docs-new/assets/sass/_mixins.scss

```diff
@@ -1,4 +1,8 @@
 @import 'foundation-sites/scss/util/mixins';
+@import "vendor/fontawesome/fontawesome.scss";
+@import "vendor/fontawesome/solid.scss";
+@import "vendor/fontawesome/icons.scss";
+@import "vendor/fontawesome/icons.scss";
```

Comment:
I don't think this is necessary. We're already bringing in fontawesome with NPM so those resources should be available without this.

Reply:
Also this section is for mixins, so it's not the right place to include these anyway.